src/applications/static-pages/health-care-manage-benefits/get-medical-records-page/components/AuthContent/index.js

```diff
@@ -2,6 +2,22 @@
 import React from 'react';
 // Relative imports.
 import CallToActionWidget from 'platform/site-wide/cta-widget';
+import CernerCallToAction from '../../../components/CernerCallToAction';
+
+const callToActions = [
+  {
+    deriveHeaderText: facilityNames =>
+      `Get your medical record from ${facilityNames}`,
+    href: '',
+    label: 'Get record on My VA Health',
+  },
+  {
+    deriveHeaderText: () =>
+      `Get your medical record from another VA Medical Center`,
+    href: '',
```

Comment:
Need the 2 `href` links here:

![image](https://user-images.githubusercontent.com/12773166/86203832-35422c00-bb23-11ea-968d-b357cbf5f192.png)

Reply:
I've requested these from Cerner, but haven't received yet.

Reply:
Thank you! 🙂